### PR TITLE
Us26 s02 navbar

### PR DESCRIPTION
**Pull request by sorrento974**

### CODE REVIEW

File: client/src/App.css [hunk 1]
```diff
@@ -2,6 +2,7 @@
   padding: 0;
   margin: 0;
   box-sizing: border-box;
+  /* background-color: var(--secondary-color); */
```
Comment (Delgorithm) [inline client/src/App.css]: You probably delete it since its a comment

File: client/src/App.css [hunk 2]
```diff
@@ -14,3 +15,6 @@
   --font-subtitle: "Sedan", sans-serif;
   --font-content: "Inter", sans-serif;
 }
+
```
Comment (Delgorithm) [inline client/src/App.css]: Blank spaces ?

File: client/src/App.jsx [hunk 3]
```diff
@@ -1,10 +1,15 @@
 import "./App.css";
-import Home from "./pages/Home";
+import Navbar from "./components/NavBar";
+// import SearchBar from "./components/SearchBar";
```
Comment (Delgorithm) [inline client/src/App.jsx]: You should delete this comment

File: client/src/App.jsx [hunk 4]
```diff
@@ -1,10 +1,15 @@
...
+
+
 
 function App() {
   return (
-    <main>
-      <Home />
+    <main className="content">
+      {/* <SearchBar /> */}
```
Comment (Delgorithm) [inline client/src/App.jsx]: Another comments to delete

File: client/src/assets/images/gorille-art-bdx-4.jpg
Comment (Delgorithm) [inline client/src/assets/images/gorille-art-bdx-4.jpg:1]: Why did you add many photos from the same street art ? Is it for exemple for the DB ?

File: client/src/components/NavBar.css [hunk 5]
```diff
@@ -0,0 +1,68 @@
+.navbar-desktop,
+.navbar-mobile {
+  position: fixed;
+  width: 100%;
+  height: 5rem;
+  /* background-color: var(--third-color); */
```
Comment (Delgorithm) [inline client/src/components/NavBar.css]: Comment to delete

File: client/src/components/NavBar.jsx [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+          </li>
+          <li className="navbar-item">
+            <NavLink to="/profile">
+              <BsPersonCircle className="navbar-link-mobile" />
+            </NavLink>
+          </li>
+        </ul>
+      </nav>
+      <nav className="navbar-desktop">
+        <img className="logo" src="../src/assets/images/LOGO PH.jpg" alt="logo pixhunter" />
```
Comment (Delgorithm) [inline client/src/components/NavBar.jsx]: It's good. But you should rename your file "LOGO PH.jpg" to "logo_ph.jpeg" or just "logo.jpeg"

Furthermore, you should go for a "png" or "favicon" instead of "jpeg"

File: client/src/components/SearchBar.css [hunk 7]
```diff
@@ -0,0 +1,36 @@
+/* .search-input {
```
Comment (Delgorithm) [inline client/src/components/SearchBar.css]: What the hell ?

File: client/src/components/SearchBar.jsx [hunk 8]
```diff
@@ -0,0 +1,8 @@
+// import "./SearchBar.css"
```
Comment (Delgorithm) [inline client/src/components/SearchBar.jsx]: hm You really need to delete those comments

File: client/src/main.jsx [hunk 9]
```diff
@@ -3,11 +3,16 @@ import ReactDOM from "react-dom/client";
 
 import { createBrowserRouter, RouterProvider } from "react-router-dom";
 
+import Homepage from "./pages/Home";
 import App from "./App";
 
 const router = createBrowserRouter([
   {
     path: "/",
+    element: <Homepage />,
```
Comment (Delgorithm) [inline client/src/main.jsx]: Since we have the page "Home" you should consider to rename this component with something more suitable : "welcoming" or "reception" for example

File: client/src/pages/Home.jsx [hunk 10]
```diff
@@ -1,6 +1,21 @@
+import "../styles/Home.css";
+import { NavLink } from "react-router-dom";
+// import {BsArrowRight} from "react-icons/bs"
```
Comment (Delgorithm) [inline client/src/pages/Home.jsx]: Comment to delete

File: client/src/pages/Home.jsx [hunk 11]
```diff
@@ -1,6 +1,21 @@
...
+
 function Home() {
     return (
-        <p>Page d'accueil</p>
+        <>
+        {/* <p>Page d'accueil</p> */}
```
Comment (Delgorithm) [inline client/src/pages/Home.jsx]: Comment to delete

File: client/src/pages/Home.jsx [hunk 12]
```diff
@@ -1,6 +1,21 @@
...
+        
+        <img className="gorille-art" src="../src/assets/images/gorille-art-bdx-2.jpg" alt="gorille-art" />
+        <h1 className="title-site">PixHunt</h1>
+        <p className="welcome-sentence">La chasse au <span className="street-art">Street Art</span> peut commencer !</p>
+        {/* <BsArrowRight className="bs-arrow-right"/> */}
```
Comment (Delgorithm) [inline client/src/pages/Home.jsx]: Comment to delete

File: client/src/pages/Home.jsx [hunk 13]
```diff
@@ -1,6 +1,21 @@
...
+        <NavLink to="/mappage" ><button className="formButton" type="submit">
```
Comment (Delgorithm) [inline client/src/pages/Home.jsx]: It is useless to add a "button" to a "NavLink". You can add a className directly in the "NavLink": like that 
<NavLink to="/mappage" className="x" />...

Buttons are not made for this kind of method

File: client/src/styles/Home.css [hunk 14]
```diff
@@ -0,0 +1,60 @@
...
+  box-shadow: 0px 8px 40px var(--fourth-color);
+}
+.title-site {
+  font-family: var(--font-title);
+  text-align: center;
+  padding: 1rem 0.5rem;
+  font-weight: bold;
+  font-size: 3rem;
+}
+/* .bs-arrow-right {
```
Comment (Delgorithm) [inline client/src/styles/Home.css]: Comment to delete

**Review by Delgorithm — CHANGES_REQUESTED** [review]

**Review by Delgorithm — APPROVED** [review]

File: client/src/components/NavBar.jsx [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+            <NavLink to="/profile">
+              <BsPersonCircle className="navbar-link-mobile" />
+            </NavLink>
+          </li>
+        </ul>
+      </nav>
+      <nav className="navbar-desktop">
+        <img
+          className="logo"
+          src="../src/assets/images/LOGO PH.jpg"
```
Comment (Delgorithm) [inline client/src/components/NavBar.jsx]: You should rename the file to "logo.jpg"
Even better, you should go for a "logo.png"

File: client/src/styles/NavBar.css [hunk 16]
```diff
@@ -0,0 +1,72 @@
+.navbar-desktop,
+.navbar-mobile {
+  position: fixed;
+  width: 100%;
+  height: 5rem;
+  /* background-color: var(--third-color); */
```
Comment (Delgorithm) [inline client/src/styles/NavBar.css]: You should delete this comment

File: client/src/styles/NavBar.css [hunk 17]
```diff
@@ -0,0 +1,72 @@
...
+  box-shadow: 0 2px 5px var(--fourth-color);
+  z-index: 1;
+}
+
+.navbar-list-mobile {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  list-style: none;
+  padding: 10px 0;
```
Comment (Delgorithm) [inline client/src/styles/NavBar.css]: You should go for "rem" instead of "px"

File: client/src/styles/NavBar.css [hunk 18]
```diff
@@ -0,0 +1,72 @@
...
+}
+.navbar-link-mobile {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: var(--fourth-color);
+  font-size: 35px;
```
Comment (Delgorithm) [inline client/src/styles/NavBar.css]: You should go for "rem" instead of "px"

**Review by Delgorithm — CHANGES_REQUESTED** [review]

**Review by Delgorithm — APPROVED** [review]

File: client/src/styles/NavBar.css [hunk 19]
```diff
@@ -0,0 +1,73 @@
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  position: relative;
+  width: 101.5%;
```
Comment (Dinosaure7) [inline client/src/styles/NavBar.css:5]: 101.5% set to correct an issue with the map being too large

File: client/src/styles/NavBar.css [hunk 20]
```diff
@@ -0,0 +1,73 @@
...
+
+.navList {
+  color: var(--third-color);
+}
+@media (min-width: 768px) {
+  .navText,
+  .logo {
+    top: 0;
+    bottom: auto;
+
```
Comment (Dinosaure7) [inline client/src/styles/NavBar.css:39]: Line space unwanted for

**Review by Dinosaure7 — APPROVED** [review]
PR is OK, only a few simple issues in CSS

**Review by KevinDavoust — APPROVED** [review]
